File: Python/PRAK601-2210817210013-Muhammad-Shandy-Dailami.py
for x in range(3):
    baris, kolom = map(int, input().split())
    angka = []
    x = 0
    isi = list(map(int, input().split()))
    print ()
    for i in range(baris):
        angka.append(isi[x : x + kolom])
        x += kolom
    for baris in angka:
        for isi in baris:
            print(isi, end=' ')
        print()
    print()

File: Bahasa C/PRAK601-2210817210013-Muhammad-Shandy-Dailami.c
#include <stdio.h>

int main(){
    int x = 0;
    do{
        int baris, kolom, isi;
        scanf("%d %d", &baris, &kolom);
        int angka[baris][kolom];
        for (int i = 0; i < baris; i++){
            for (int j = 0; j < kolom; j++){
                scanf("%d", &isi);
                angka[i][j] = isi;
            }
        }
        printf("\n");
        for (int i = 0; i < baris; i++){
            for (int j = 0; j < kolom; j++){
                printf("%d ", angka[i][j]);
            }
            printf("\n");
        }
    x++;
    }while(x<=2);
    return 0;
}